pyproject.toml

```diff
@@ -248,6 +248,9 @@ select = [
 
   # (invalid escape sequence) flag errant backslashes
   "W605",
+
+  # pyupgrade - automatically upgrade Python syntax to newer versions
+  "UP",
```

Comment:
nit: Please follow comment formatting above (parens).

Also specify how this determines what version to target. I believe this comes from the ruff `target-version` setting, which seems to still be on 3.8.

Reply:
You're right, didn't notice we haven't upgraded it. Setting to `3.9`.